src/main/scala/mesosphere/marathon/event/http/HttpEventStreamHandleActor.scala

```diff
@@ -0,0 +1,45 @@
+package mesosphere.marathon.event.http
+
+import java.io.{ EOFException, IOException }
+
+import akka.actor.Actor
+import akka.event.EventStream
+import mesosphere.marathon.api.v2.json.Formats._
+import mesosphere.marathon.event.{ EventStreamAttached, EventStreamDetached, MarathonEvent }
+import org.apache.log4j.Logger
+import play.api.libs.json.Json
+
+import scala.util.Try
+
+class HttpEventStreamHandleActor(handle: HttpEventStreamHandle, stream: EventStream) extends Actor {
+
+  private[this] val log = Logger.getLogger(getClass.getName)
+
+  override def preStart(): Unit = {
+    stream.subscribe(self, classOf[MarathonEvent])
+    stream.publish(EventStreamAttached(handle.remoteAddress))
+  }
+
+  override def postStop(): Unit = {
+    log.info(s"Stop actor $handle")
+    stream.unsubscribe(self)
+    stream.publish(EventStreamDetached(handle.remoteAddress))
+    Try(handle.close()) //ignore, if this fails
+  }
+
+  override def receive: Receive = {
+    case event: MarathonEvent =>
+      try {
+        //this can cause an IOException
+        handle.sendMessage(Json.stringify(eventToJson(event)))
```

Comment:
Mmmh - I will make this call an async future, count the outstanding ones and drop messages, if a limit is reached.